variables.tf

```diff
@@ -103,7 +103,7 @@ variable "timeout_seconds" {
 variable "service_account_email" {
   type        = string
   description = "Service Account email needed for the service"
-  default     = null
+  default     = ""
```

Comment:
From our offline conversation, the reason for this change is because the registry identifies these as required inputs although they are not.
![image](https://user-images.githubusercontent.com/10992720/126360551-38050a8a-a647-4c21-a911-5f5db6fb5260.png)